apps/src/aiTutor/chatApi.ts

```diff
@@ -51,15 +54,15 @@ const formatForChatCompletion = (
 export async function getChatCompletionMessage(
   systemPrompt: string,
   userMessageId: number,
-  newMessage: string,
+  formattedQuestion: string,
```

Comment:
I changed this name to make it a little more apparent that this is getting wrapped with some additional context (in this case, with the student code). Before, that context was being included in the system prompt instead of in the question itself.